src/Fixer/Phpdoc/PhpdocTypesFixer.php

```diff
@@ -132,7 +132,7 @@ public function getPriority(): int
     protected function normalize(string $type): string
     {
         return Preg::replaceCallback(
-            '/(\b|(?=\$|\\\\))(\$|\\\\)?'.TypeExpression::REGEX_IDENTIFIER.'(?!\\\\|\h*:)/',
+            '/(\b|(?=\$|\\\\))(\$|\\\\)?'.TypeExpression::REGEX_IDENTIFIER.'(?!\\\\|\'|\h*:)/',
```

Comment:
The same problem might be with `"` quoted strings. But I think we should parse the type using TypeExpression.

Reply:
That's what I thought too. It would be better to parse types and handle them separately (each on its own), rather than treating whole type declaration as one string.

Reply:
I never saw a literal-string phpdoc with double-quotes, but if you already have something different in mind to resolve the issue (and maybe even more issues) then I will close this PR.

Reply:
@voku don't give up, maybe you want to give it a try and introduce proper type parsing here 😉. @mvorisek already gave a clue.

Reply:
In https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7108 I have coded impl. based on `TypeExpression`, not so easy, as identifiers are currently not parsed as subexpressions.